common/ManagementTestShared/ManagementRecordedTestBase.cs

```diff
@@ -7,7 +7,7 @@
 
 using Azure.Core;
 using Azure.Core.TestFramework;
-using Azure.Management.Resources;
+using Azure.ResourceManager.Resources;
```

Comment:
We can't simply change in this way as it is shared by multiple projects. Please change as below

```suggestion
#if RESOURCES_RP
using Azure.ResourceManager.Resources;
#else
using Azure.Management.Resources;
#endif
```

Reply:
Done